.gitignore

```diff
@@ -1,2 +1,3 @@
 _build*
 _build_html*
+**/*.swp
```

Comment:
Since I've never seen vim crash, I've had this in my`.vimrc` for years: `set noswapfile` :smile_cat:

Reply:
vim may not crash, but my laptop sure does...